src/PrestaShopBundle/Resources/views/Admin/TwigTemplateForm/form_div_layout.html.twig

```diff
@@ -346,9 +346,21 @@
 
   {% for child in form -%}
     {% if not child.rendered %}
-      {{- ps.form_group_row(child, { attr: child.vars.attr }, {
-        'label': child.vars.label
-      }) -}}
+      {% if 'hidden' in child.vars.block_prefixes %}
+        {{- form_row(child) -}}
+      {% else %}
+        {% if '_supplier_id_state' in child.vars.block_prefixes %}
+          <div class="js-supplier-state">
+            {{- ps.form_group_row(child, { attr: child.vars.attr }, {
+              'label': child.vars.label
+            }) -}}
+          </div>
+        {% else %}
+          {{- ps.form_group_row(child, { attr: child.vars.attr }, {
+            'label': child.vars.label
+          }) -}}
+        {% endif %}
+      {% endif %}
```

Comment:
Since agreed to use form_widget - this needs to be reworked for form_widget block. That's why have this pull request still in WIP.